### ISSUE TITLE
Publish artifacts to a maven repo, to allow for easier workflows.

### ISSUE THREAD

**Issue by me4502** [phr00t/jmonkeyengine-virtual-reality#3]
Currently downloading jars is nowhere near as easy as using a maven/gradle workflow.

**Comment by phr00t** [comment]
I agree... just don't have the time to set it up at the moment. When I do, I'll followup on this.

**Comment by jherico** [comment]
I have an existing central-repository setup for Jovr.  If you want I can push the jars/source up under the org.saintandreas group ID as a temporary measure till you get access to a jmonkey group ID.

**Comment by phr00t** [comment]
I presume jMonkeyVR should have its own... not sure jMonkeyVR belongs with the jMonkeyEngine group ID (jMonkeyVR integrates with jMonkeyEngine, but it isn't considered an "official" library). I'm working on some bugs with 5089 at the moment... not sure how to go about setting this up (or even needing a "group ID" was a thing). If someone else wants to go about setting this up, I'd be more than happy to allow you to do so.

**Comment by jherico** [comment]
The workflow for getting stuff published to the central maven repository is kind of a hassle involving a number of steps, the first of which is to get a group id.  I just have the advantage of having done it before.  I'll try to set up a group ID for jmonkeyvr, although it's possible it might get rejected for not following the common naming convention of something like `org.jmonkeyvr`

**Comment by phr00t** [comment]
Thank you @jherico, much appreciated.

**Comment by jherico** [comment]
https://issues.sonatype.org/browse/OSSRH-21474

per their guidelines, it looks like until there's a better home for it, the group id for maven will be `com.github.phr00t`

**Comment by phr00t** [comment]
Whatever works! :smile: Thank you for following up on this!

**Comment by jawaff** [comment]
Are you guys still working on getting this project into maven's repos? I'm not really sure how the procedure works, but it looks like the sonatype guys have everything ready. @jherico What other hoops are left to jump through in order to make this happen?

**Comment by phr00t** [comment]
I am not working on getting this project to maven specs. I just do not have time & only maintain this project for my games. Feel free to fork this project (updating to the latest jMonkeyEngine would be nice, too).
